lib/led/rgb.js

```diff
@@ -488,7 +488,7 @@ class RGB {
    */
 
   [Animation.render](frames) {
-    return this.color(frames[0]);
+    return this.update(frames[0]);
```

Comment:
The `Animation.render` method here was calling `color()`, which does a lot of normalization and validation before itself calling `update` on line 270 above. 
Now the render method behaves more similar to render in the [`led class`](https://github.com/code-dot-org/johnny-five/blob/812e095ffa2f8c0537d4796422d619d2950d4141/lib/led/led.js#L334) which 'trusts' its animations and calls `update` directly - this is similar to @islemaster 's implementation [here](https://github.com/code-dot-org/johnny-five/pull/10#discussion_r160807279).
This also bypasses the reassignment of `state.prev` to a 'tween' value when `Animation.render` is called for `pulse` function.